### PR TITLE
feat(cli): lacework policy command

### PR DESCRIPTION
**Pull request by hazedav**
Building out lacework policy command functionality.

Includes:
- `lacework policy list`
- `lacework policy show`
- `lacework policy create`
- `lacework policy update`
- `lacework policy delete`

ALLY-422

### CODE REVIEW

**Comment by hazedav** [comment]
This is final review of work that had individual reviews per each of the subtasks under ALLY-422.  Thanks!

File: api/api.go [hunk 1]
```diff
@@ -85,6 +85,9 @@ const (
 	apiLQLDescribe    = "v1/external/lql/describe"
 	apiLQLQuery       = "v1/external/lql/query"
 
+	// Custom Policies
+	apiPolicy = "external/lqlPolicies"
```
Comment (afiune) [inline api/api.go:89]:
```suggestion
	apiPolicy = "v1/external/lqlPolicies"
```

**Review by afiune — CHANGES_REQUESTED** [review]
only one single blocker, the rest looks great!

File: api/api.go (hunk 1 above)
Comment (hazedav) [reply api/api.go:89]: Fixed.

**Review by afiune — APPROVED** [review]
tenor-46500580

**Pull request merged**